src/DB/DumpGenerator.php

```diff
@@ -7,19 +7,52 @@
 
 namespace Magento\MagentoCloud\DB;
 
+use Magento\MagentoCloud\Config\ConfigException;
+use Magento\MagentoCloud\Config\Database\DbConfig;
+use Magento\MagentoCloud\DB\Data\ConnectionInterface;
 use Magento\MagentoCloud\Filesystem\DirectoryList;
+use Magento\MagentoCloud\Package\UndefinedPackageException;
 use Magento\MagentoCloud\Shell\ShellInterface;
 use Psr\Log\LoggerInterface;
+use Magento\MagentoCloud\DB\Data\ConnectionFactory;
 
 /**
  * Creates database dump and archives it
  */
 class DumpGenerator
 {
     /**
-     * Template for dump file name where %s should be changed to timestamp for uniqueness
+     * Database names
      */
-    const DUMP_FILE_NAME_TEMPLATE = 'dump-%s.sql.gz';
+    const DATABASE_MAIN = 'main';
+    const DATABASE_QUOTE = 'quote';
```

Comment:
You already added 'quote' and 'sales' constants in DeployInterface is better not duplicate constants with the same value.  `const DATABASE_QUOTE = DeployInterface::SPLIT_DB_VALUE_QUOTE;`

Reply:
@oshmyheliuk These constants have different contexts